rmw_fastrtps_cpp/src/functions.cpp

```diff
@@ -1525,8 +1528,8 @@ rmw_ret_t rmw_destroy_subscription(rmw_node_t * node, rmw_subscription_t * subsc
     }
     delete info;
   }
-
-  delete (subscription);
+  rmw_free(const_cast<char *>(subscription->topic_name));
```

Comment:
should `subscription->topic_name` be `nullptr` after being freed like its done for the publisher ?